### PR TITLE
Preliminary testing setup/functionality

### PR DESCRIPTION
**Pull request by clmcguire**
Trying to get some basic tests up for each component, will try to add more functionality- feel free to send me suggestions or any feedback.

### CODE REVIEW

File: src/Create/Create.test.tsx [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
-describe("Create", () => {
-  test("matches snapshot", () => {
-    const wrapper = shallow(<Create />);
-    expect(wrapper).toMatchSnapshot();
-  });
-});
+describe("Render Create component", () => {
+    test("matches snapshot", () => {
+        
+        const {asFragment} = render(<JournalProvider><MemoryRouter><Create /></MemoryRouter></JournalProvider>); //fragment is portion of what user sees
```
Comment (DAC098) [inline src/Create/Create.test.tsx]: to keep thing consistent with the app, try to use the `HashRouter`

File: src/Home/Home.test.tsx [hunk 2]
```diff
@@ -1,10 +1,28 @@
...
-import { shallow } from "enzyme";
+import{render, screen} from "@testing-library/react";
 import Home from "./Home";
+import { JournalProvider } from "../JournalContext";
+import { MemoryRouter } from "react-router-dom";
+import "@testing-library/jest-dom";
 
+describe("Home renders correctly", () => {
+    test("displays calendar and headers", () => {
+        render(<JournalProvider><MemoryRouter><Home /></MemoryRouter></JournalProvider>)
```
Comment (DAC098) [inline src/Home/Home.test.tsx]: swap over to the `HashRouter`

File: src/Navbar/Navbar.test.tsx [hunk 3]
```diff
@@ -1,10 +1,25 @@
...
-describe("Navbar", () => {
-  test("matches snapshot", () => {
-    const wrapper = shallow(<Navbar />);
-    expect(wrapper).toMatchSnapshot();
-  });
+
+describe("Navbar renders properly", () => {
+    test("navbar main links exist", () => {
+        render(
+        <BrowserRouter>
```
Comment (DAC098) [inline src/Navbar/Navbar.test.tsx]: swap to use the `HashRouter`

**Review by DAC098 — CHANGES_REQUESTED** [review]

**Comment by DAC098** [comment]
@clmcguire did you want to close the pull request?

**Comment by clmcguire** [comment]
@DAC098 I did not but I submitted a new one with the changes you requested- does that work?

**Comment by DAC098** [comment]
it is fine, I noticed the other one and looking at it

**Pull request closed without merging**